identity/tests/test_data.sql

```diff
@@ -0,0 +1,22 @@
+INSERT INTO users (sub, email, given_name, family_name, picture, oauth_access_token, oauth_access_token_valid, oauth_refresh_token, active) VALUES ('5', 'richard.henderson@example.net', 'Richard', 'Henderson', 'https://example.com/avatar.jpg', 'access_token', '2020-12-31 24:00:00', 'refresh_token', true);
+INSERT INTO users (sub, email, given_name, family_name, picture, oauth_access_token, oauth_access_token_valid, oauth_refresh_token, active) VALUES ('6', 'olivia.springer@example.net', 'Olivia', 'Springer', 'https://example.com/avatar.jpg', 'access_token', '2020-12-31 24:00:00', 'refresh_token', true);
+INSERT INTO users (sub, email, given_name, family_name, picture, oauth_access_token, oauth_access_token_valid, oauth_refresh_token, active) VALUES ('7', 'alexander.carr@example.net', 'Alexander', 'Carr', 'https://example.com/avatar.jpg', 'access_token', '2020-12-31 24:00:00', 'refresh_token', true);
+INSERT INTO users (sub, email, given_name, family_name, picture, oauth_access_token, oauth_access_token_valid, oauth_refresh_token, active) VALUES ('8', 'yvonne.thomson@example.net', 'Yvonne', 'Thomson', 'https://example.com/avatar.jpg', 'access_token', '2020-12-31 24:00:00', 'refresh_token', true);
+INSERT INTO users (sub, email, given_name, family_name, picture, oauth_access_token, oauth_access_token_valid, oauth_refresh_token, active) VALUES ('9', 'alan.hemmings@example.net', 'Alan', 'Hemmings', 'https://example.com/avatar.jpg', 'access_token', '2020-12-31 24:00:00', 'refresh_token', true);
+INSERT INTO users (sub, email, given_name, family_name, picture, oauth_access_token, oauth_access_token_valid, oauth_refresh_token, active) VALUES ('10', 'eric.vance@example.net', 'Eric', 'Vance', 'https://example.com/avatar.jpg', 'access_token', '2020-12-31 24:00:00', 'refresh_token', true);
+INSERT INTO users (sub, email, given_name, family_name, picture, oauth_access_token, oauth_access_token_valid, oauth_refresh_token, active) VALUES ('11', 'audrey.miller@example.net', 'Audrey', 'Miller', 'https://example.com/avatar.jpg', 'access_token', '2020-12-31 24:00:00', 'refresh_token', true);
+INSERT INTO users_roles (user_id, role_id) VALUES (1, 1);
+INSERT INTO users_roles (user_id, role_id) VALUES (2, 1);
+INSERT INTO users_roles (user_id, role_id) VALUES (3, 1);
+INSERT INTO users_roles (user_id, role_id) VALUES (4, 1);
+INSERT INTO users_roles (user_id, role_id) VALUES (5, 1);
+INSERT INTO users_roles (user_id, role_id) VALUES (6, 1);
+INSERT INTO users_roles (user_id, role_id) VALUES (7, 1);
+INSERT INTO users_roles (user_id, role_id) VALUES (8, 2);
+INSERT INTO users_roles (user_id, role_id) VALUES (9, 2);
+INSERT INTO users_roles (user_id, role_id) VALUES (10, 3);
+INSERT INTO users_roles (user_id, role_id) VALUES (11, 1);
```

Comment:
Correct, that would work as well. I just like it explicit, because if you have a wrong value in between the error message tells you only about the line and not really anything about the value.